tests/test_codex_feed_hooks.py

```
@@ -377,13 +380,59 @@ def test_codex_monitor_exits_when_workspace_has_no_surfaces(cli_path: str, root:
+    socket_path = root / "cmux-monitor-dead-owner.sock"
+    transcript_path = root / "codex-session-dead-owner.jsonl"
+    transcript_path.write_text("", encoding="utf-8")
+
+    session_id = f"codex-monitor-dead-owner-session-{os.getpid()}"
+    env = os.environ.copy()
+    env["CMUX_SOCKET_PATH"] = str(socket_path)
+    env["CMUX_WORKSPACE_ID"] = "workspace-codex-feed-test"
+
+    with FakeCmuxSocket(socket_path, None) as fake:
+        try:
+            result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)